lib/schemas/FieldSchema.js

```diff
@@ -101,6 +101,10 @@ module.exports = makeSchema({
       description: 'Does the value of this field affect the definitions of other fields in the set?',
       type: 'boolean',
     },
+    inputFormat: {
+      description: 'Really only useful for subdomain type. This allows the user to specify a comma-separated string like "https://,{{input}},.yourdomain.com/".',
+      type: 'string',
```

Comment:
If we can make this `type: 'subdomain'`, we should be able to add some validation where it then requires `inputFormat`. Do you think it's an overkill to have it that much explicit?